corehq/tests/nose.py

```diff
@@ -297,11 +297,12 @@ def teardown(self):
 
         self.blob_db.close()
 
+        self.delete_elastic_indexes()
```

Comment:
How much time does this typically take? If it takes significant time (or if it results in significant database setup time in the next test run) we may want to revert 179195d3d700587390aad5270c956304599d1b8a and find a different solution.

Another reason we may want to find a different solution would be if this causes unintuitive behavior for developers because `reusedb` has inconsistent behavior for different storage backends.

Reply:
I would prefer not reverting it because
1. deleting ES indexes is pretty fast, for this case on my local system deleting of every index just took .2 seconds.
2. Setup and deleting of every ES index happens automatically by @es_test decorator. 

So, I think this optimisation would not benefit us in similar terms as it does in couch and sql.